demisto_sdk/json_to_outputs/json_to_outputs.py

```diff
@@ -0,0 +1,233 @@
+                "key":"new",
+                "colorName":"blue-gray",
+                "name":"To Do"
+        }
+    }
+}
+
+==>
+
+arguments: []
+name: integration-command
+outputs:
+- contextPath: Demisto.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Description
+  description: ''
+  type: String
+- contextPath: Demisto.Summary
+  description: ''
+  type: String
+- contextPath: Demisto.Created
+  description: ''
+  type: String
+- contextPath: Demisto.Issuetype.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Issuetype.Name
+  description: ''
+  type: String
+- contextPath: Demisto.Project.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Project.Key
+  description: ''
+  type: String
+- contextPath: Demisto.Project.ProjectTypeKey
+  description: ''
+  type: String
+- contextPath: Demisto.Status.Id
+  description: ''
+  type: Number
+- contextPath: Demisto.Status.StatusCategory.Key
+  description: ''
+  type: String
+- contextPath: Demisto.Status.StatusCategory.Colorname
+  description: ''
+  type: String
+- contextPath: Demisto.Status.StatusCategory.Name
+  description: ''
+  type: String
+"""
+
+
+def input_multiline():
+    sentinel = ''  # ends when this string is seen
+    return '\n'.join(iter(input, sentinel))
+
+
+def flatten_json(nested_json, camelize=False):
+    out = {}
+
+    def flatten(x, name=''):
+        # capitalize first letter in each key
+        try:
+            name = name[0].upper() + name[1:] if camelize else name
+        except IndexError:
+            name = name.title() if camelize else name
+
+        if isinstance(x, dict):
+            for a in x:
+                flatten(x[a], name + a + '.')
+        elif isinstance(x, list):
+            for a in x:
+                flatten(a, name[:-1] + '.')
+        else:
+            out[name.rstrip('.')] = x
+
+    flatten(nested_json)
+    return out
+
+
+def jsonise(context_key, value, description=''):
+    return {
+        'contextPath': context_key,
+        'description': description,
+        'type': determine_type(value)
+    }
+
+
+def is_date(val):
+    """
+    Determines if val is Date, if yes returns True otherwise False
+    """
+    if isinstance(val, (int, float)) and val > 15737548065 and val < 2573754806500:
+        # 15737548065 is the lowest timestamp that exist year - 1970
+        # 2573754806500 is the year 2050 - I believe no json will contain date time over this time
+        # if number is between these two numbers it probably is timestamp=date
+        return True
+
+    if isinstance(val, str) and len(val) >= 10 and len(val) <= 30 and dateparser.parse(val):
+        # the shortest date string is => len(2019-10-10) = 10
+        # The longest date string I could think of wasn't of length over len=30 '2019-10-10T00:00:00.000 +0900'
+        # To reduce in performance of using dateparser.parse,I
+        return True
+
+    return False
+
+
+def determine_type(val):
+    if is_date(val):
```

Comment:
what id the number will be 15737548066 then we will identify that as date instead of number, not suer this will work well.

Reply:
It will work in 98% of the time. Eventually it generates YML, that user can modify any time.
If that field is not date but number he will change it